package.json

```diff
@@ -18,8 +18,8 @@
   },
   "devDependencies": {
     "edx-custom-a11y-rules": "0.1.2",
-    "pa11y": "3.6.0",
```

Comment:
`npm install --save` does this automatically, I'm more confused how they got *out* of alphabetical order in the first place.